lsp-mssql.el

```diff
@@ -243,11 +248,16 @@ PARAMS the params."
   "Result set complete handler.
 WORKSPACE is the active workspace.
 PARAMS the params."
-  (-let* ((marker (lsp-mssql-with-result-buffer
+  (-let* ((column-info (gethash "columnInfo" (gethash "resultSetSummary" params)))
```

Comment:
FWIW here we should start defining interfaces and using lsp-protocol.el otherwise it won't work with plists. I will let this in due to the fact that we are still using gethash and related on other places.